### PR TITLE
♻️More attributes added in webengage amp-analytics vendor config

### PR DESCRIPTION
**Pull request by ayushgupta29**
This PR adds more params in the requests being sent by WebEngage vendor in amp-analytics component.

### CODE REVIEW

**Comment by ayushgupta29** [comment]
@micajuine-ho Can you please take a look at this PR?

File: extensions/amp-analytics/0.1/test/vendor-requests.json [hunk 1]
```diff
@@ -490,8 +490,8 @@
     "event": "https://tags-dmp.vpadn.com/et?t=_timestamp_&sdkn=j&sdkv=1.2.0&lk=!licence_key&en=UTF-8&ctid=_client_id(vpadn-ctid)_&ev=!ev_name&pl=!payload&is_amp=1&page_id=_page_view_id_"
   },
   "webengage": {
-    "base": "https://c.!region.webengage.com/amp?licenseCode=!licenseCode&luid=_client_id(we_luid)_&pageUrl=_canonical_url_&pageTitle=_title_&referrer=_document_referrer_&vh=_viewport_height_&vw=_viewport_width_&category=application&isAmp=1",
-    "wePageview": "https://c.!region.webengage.com/amp?licenseCode=!licenseCode&luid=_client_id(we_luid)_&pageUrl=_canonical_url_&pageTitle=_title_&referrer=_document_referrer_&vh=_viewport_height_&vw=_viewport_width_&category=application&eventName=Page Viewed&isAmp=1"
+    "base": "https://c.!region.webengage.com/amp?licenseCode=!licenseCode&luid=_client_id(we_luid)_&pageUrl=_source_url_&pageTitle=_title_&referrer=_document_referrer_&vh=_viewport_height_&vw=_viewport_width_&category=application&isAmp=1&amp-viewer=_viewer_",
```
Comment (micajuine-ho) [inline extensions/amp-analytics/0.1/test/vendor-requests.json]: If you change the change to `pageUrl=_source_url_`, then you'll also have to change the request in `webengage.json`.

Comment (ayushgupta29) [reply extensions/amp-analytics/0.1/test/vendor-requests.json]: @micajuine-ho Missed it somehow, added in the new commit.

**Comment by ayushgupta29** [comment]
@micajuine-ho Will you please merge this PR?

**Pull request merged**